docs/extras/expression_language/why.mdx

```diff
@@ -0,0 +1,9 @@
+# Why use LCEL?
+
+The LangChain Expression Language was designed from day 1 to **support putting prototypes in production, with no code changes**, from the simplest “prompt + LLM” chain to the most complex chains (we’ve seen folks successfully running in production LCEL chains with 100s of steps). To highlight a few of the reasons you might want to use LCEL:
+
+- first-class support for streaming: when you build your chains with LCEL you get the best possible time-to-first-token (time elapsed until the first chunk of output comes out). For some chains this means eg. we stream tokens straight from an LLM to a streaming output parser, and you get back parsed, incremental chunks of output at the same rate as the LLM provider outputs the raw tokens. We’re constantly improving streaming support, recently we added a [streaming JSON parser](https://twitter.com/LangChainAI/status/1709690468030914584), and more is in the works.
```

Comment:
Ah we don't have the streaming JSON parser yet 😬

Reply:
Ah sorry, I saw the tweet referenced `class JsonFunctionsOutputParser` and assumed that's what this was

Reply:
Dropped